### DISCUSSION TITLE
Actual Budget

### DISCUSSION THREAD

**Discussion by tteck** [tteck/Proxmox#2727] [Announcements]
![Screenshot 2024-04-03 5 18 03 PM](https://github.com/tteck/Proxmox/assets/21342670/6349d590-4bfa-46a0-a0cb-3b4b3c566f91)

![Screenshot 2024-04-03 5 18 41 PM](https://github.com/tteck/Proxmox/assets/21342670/f5370ac3-a1f1-4f9d-962a-69c38e0a2b82)

**Comment by buczi11** [comment]
![319449454-9372a1ef-a85c-46e9-b77e-5c3404099146](https://github.com/tteck/Proxmox/assets/6137813/309b8110-f9fc-4299-9056-2421991c3dc1)

![319449455-1fde4049-32fd-4634-9038-ad76e49df6b4](https://github.com/tteck/Proxmox/assets/6137813/3caca057-a562-4791-9df9-3dd0eb8cc53e)

👍 Work, no problems.

**Comment by oOStroudyOo** [comment]
I've been in the market looking for a decent budget app for such a long time.

This app installs and works great for me, thank you very much.

**Comment by jigglywatts** [comment]
no us bank auto-downloads? meh

**Reply by MrSaiclops** [reply]
For $15/yr SimpleFIN Bridge can natively be used. Works flawlessly for me and I'm using a local credit union.

**Reply by jigglywatts** [reply]
didn't know about SimpleFin buried in the settings. worked for my one us bank account and two credit cards. thanks!

**Comment by Davey-Gravy** [comment]
After running the script with default settings, the web app hangs on this page for me:
<img width="1600" alt="Screenshot 2024-06-11 at 11 39 57 AM" src="https://github.com/tteck/Proxmox/assets/51386700/8d3c4e27-5064-4faa-9f96-ca18ca420dbc">

No issues with any of the other helper scripts.

**Reply by Smakodak** [reply]
Did the default installation and have the same issue.

**Reply by roshavi4ak** [reply]
you need to use a reversy proxy. Once you setup an url and use https everything works great

**Comment by alabdalla** [comment]
what are the creds 
![image](https://github.com/tteck/Proxmox/assets/168717349/9aad93b0-a246-4e78-b9a4-ad9f2bc68b41)

**Reply by remarkov** [reply]
it's `root:<password you entered during the setup>`

**Comment by GTP95** [comment]
I used the script and kept the default settings, but at the address that Actual should be available to, I actually get a page with the message "Initializing the connection to the local database...". But it never completes the initialization. How can I fix this?
Thank you for your help,

GTP

**Reply by roshavi4ak** [reply]
you need to use a reversy proxy. Once you setup an url and use https everything works great

**Comment by ooii** [comment]
Is there anyway to mount a volume with data?

**Comment by jwtoler** [comment]
Got the same error using defaults. Reverse proxy and https did indeed fix things.

Maybe directly stating this on the website and instructions would be better. Instead of linking to this issue, with no clear explanation what the link is -- until you click and read the issue on Github.
